etc/behat/services/contexts.xml

```diff
@@ -313,6 +315,12 @@
             <tag name="sylius.behat.context" />
         </service>
 
+        <service id="sylius.behat.context.ui.admin.managing_country" class="%sylius.behat.context.ui.admin.managing_country.class%" scope="scenario">
```

Comment:
In ``features`` we have ``managing_tax_categories`` - plural form. It makes more sense to me, because there are many tax_categories, not just one. How about ``managing_countries`` and ``ManagingCountriesContext``?

``managing_currencies``
``managing_base_currency``
``managing_products``
``managing_store_language``